src/server/services/registrar.js

```diff
@@ -203,8 +204,46 @@ async function getShortname(offeredTermCode, classSectionID) {
   }
 }
 
+/**
+ * Get crosslisted courses from the registrar (stored as an array of shortnames)
+ *
+ * @param {string} offeredTermCode  Term.
+ * @param {string} classSectionID   ClassID aka SRS.
+ * @returns {Array}   Returns a list of crosslisted courses.
+ */
+async function getCrosslists(offeredTermCode, classSectionID) {
+  const crosslistXml = await axios(
+    `https://webservices.registrar.ucla.edu/SRDB/SRDBWeb.asmx/getConSched?user=${process.env.REG_DB_USER}&pass=${process.env.REG_DB_PASS}&term=${offeredTermCode}&SRS=${classSectionID};`
+  );
+
+  const crosslists = [];
+  xml2js.parseString(crosslistXml.data, (err, result) => {
+    result.ArrayOfGetConSchedData.getConSchedData.forEach(async crosslist => {
+      const crosslistShortname = await registrar.getShortname(
+        crosslist.term[0],
+        crosslist.srs[0]
+      );
+      if (crosslistShortname) {
+        crosslists.push({
+          term: crosslist.term[0],
+          srs: crosslist.srs[0],
+          shortname: crosslistShortname,
+        });
+      } else {
+        registrarDebug(
+          `getShortname returned null for term ${crosslist.term[0]} and SRS ${crosslist.srs[0]}`
+        );
+      }
+    });
+  });
+  registrarDebug(`Crosslists found: ${crosslists}`);
+
+  return crosslists;
+}
```

Comment:
@kilometerskang I tested this function and it always returns an empty array even if there are crosslists.

For example if in src/server/jobs/checkRegistrarWS.js you put in this code:

```
(async () => {
  const result = await registrar.getCrosslists('20S', '250822200');

  console.log(result);
})();
```

It returns []. But it should return an array with term/srs plus shortnames 20S-BIOENGR287CM-1, 20S-COMSCI287CM-1, 20S-COMSCI187CM-1